components/server/src/ome/logic/RenderingSettingsImpl.java

```diff
@@ -1159,6 +1188,17 @@ public void setPixelsData(PixelsService dataService) {
         getBeanHelper().throwIfAlreadySet(this.pixelsData, dataService);
         pixelsData = dataService;
     }
+    
+    /**
+     * Sets injector. For use during configuration. Can only be called once.
+     * 
+     * @param rawPixStore
+     *            The value to set.
+     */
+    public void setRawPixStore(RawPixelsStore rawPixStore) {
```

Comment:
Sorry, missed that comment. I'll add another small commit for the rename of the variable to 'rawPixelsStore'.